packages/widget-v2/src/state/swapExecutionPage.ts

```diff
@@ -1,21 +1,30 @@
 import { atomWithMutation, atomWithQuery } from "jotai-tanstack-query";
 import { skipClient, skipRouteAtom } from "./skipClient";
 import { atom } from "jotai";
-import { RouteResponse, TxStatusResponse, UserAddress } from "@skip-go/client";
+import { ExecuteRouteOptions, RouteResponse, TxStatusResponse, UserAddress } from "@skip-go/client";
 import { MinimalWallet } from "./wallets";
 import { atomEffect } from "jotai-effect";
 import { atomWithStorage } from "jotai/utils";
 import { setTransactionHistoryAtom, transactionHistoryAtom } from "./history";
 import { SimpleStatus } from "@/utils/clientType";
 import { errorAtom, ErrorType } from "./errorPage";
 
+type ValidatingGasBalanceData = {
+  chainID?: string;
+  txIndex?: number;
+  status: "success" | "error" | "pending" | "completed"
```

Comment:
every time i see us adding ID it hurts me, but we'll just have to update this later once we refactor the client library